packages/client/components/EmailPasswordAuthForm.tsx

```diff
@@ -180,7 +180,11 @@ const EmailPasswordAuthForm = forwardRef((props: Props, ref: any) => {
     atmosphere.setAuthToken(token)
     if (invitationToken) {
       localStorage.removeItem(LocalStorageKey.INVITATION_TOKEN)
-      AcceptTeamInvitationMutation(atmosphere, {invitationToken}, {history, onCompleted, onError})
+      AcceptTeamInvitationMutation(
+        atmosphere,
+        {invitationToken},
+        {history, onCompleted, onError, ignoreApproval: true}
```

Comment:
`onError` already propagates the error to the form. passing in `ignoreApproval` prevents a snackbar with the same message from popping up